### PR TITLE
Use pg@6

### PR DESCRIPTION
**Pull request by marshallswain**
Addresses a problem described in #246

### CODE REVIEW

**Comment by ekryski** [comment]
Grrr.... looks to be failing on RethinkDB. Something about no connection pools. Maybe the RethinkDB just isn't ready for connections by the time the test is run.

I'll pull this branch down and manually verify that PG works at least.

**Comment by daffl** [comment]
The RethinkDB stuff just started failing a couple of days ago. I just created a follow-up issue at https://github.com/feathersjs/generator-feathers/issues/248

**Comment by daffl** [comment]
Let's skip the RethinkDB tests for now.

**Comment by ekryski** [comment]
ok PG seems to be working locally. :shipit:

File: generators/connection/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ module.exports = class ConnectionGenerator extends Generator {
       mariadb: 'mysql',
       mysql: 'mysql',
```
Comment (daffl) [inline generators/connection/index.js]: Could you also change this to `mysql2` while we're at it? That's what I got a complaint about.

**Comment by ekryski** [comment]
Woot! Thanks @marshallswain, you wanna do a new patch release?

**Comment by marshallswain** [comment]
I already did, but I guess I should follow up.  ;)  

Released as `generator-feathers@1.2.7` and `feathers-cli@2.3.2`

**Pull request merged**